Source/Blazorise/Components/Alert/Alert.razor.cs

```diff
@@ -109,7 +113,7 @@ private void HandleVisibilityState( bool visible )
         /// <param name="visible">True if <see cref="Alert"/> is visible.</param>
         private void RaiseEvents( bool visible )
```

Comment:
Possible and worth it converting?

Reply:
This one is problematic because it can be called from Visible setter. It is private so it doesn't affect users. We can leave it.